webhook-deprecations.rst

```diff
@@ -0,0 +1,41 @@
+.. post:: Jan 11, 2019
+    :tags: deprecation
+    :author: Anthony
+
+Incoming Webhook Deprecations
+=============================
+
+In the coming weeks and months, Read the Docs will be moving some projects away
+from our legacy incoming webhooks. About 2 years ago, these incoming webhooks
+were replaced with project webhook integrations, at which point we improved
```

Comment:
I think we need some explanation of the difference between "legacy webhooks" and "project webhooks". I personally am not super familiar with the difference in our code and I was a bit confused by these couple sentences.

Reply:
Agreed.

From a technical point of view, "legacy webhooks" are the URLs

* /github
* /gilab
* /build
* /bitbucket

and the new projects integrations are:

* /webhook/github
* /webhook/*

Reply:
Yeah, hard to explain at a high level. The new webhooks are attached to projects, the old webhooks were public -- so you could trigger a build for whatever project you wanted. I debate getting into a deep discussion in the blog post, but probably does warrant something in the docs maybe?

Reply:
Okay, i tried to clarify. Feel free to merge if it makes more sense now. It's rather hard to distinguish the two, we never really talked about or exposed the legacy webhooks. They were hidden and only internal.